src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java

```diff
@@ -7,4 +7,5 @@
 @Repository
 public interface UserRepository extends CrudRepository<User, Integer> {
     User findByUsername(String username);
```

Comment:
Could we use these in the confirm username/email functions?

Reply:
went ahead and removed the findByEmail, it was from something else I was trying to work on previously